FAQ.md

```diff
@@ -23,7 +23,7 @@ this.setState({ editorState: EditorState.createEmpty() })
 
 ## Why are mentions broken after using `convertFromRaw` and throwing an error?
 
-We design the API to accept an Immutable Map for a Mention. After saving your datastructure to the erver and using `convertFromRaw` the mentions in there are plain objects. I (Nik) believe this is a law in our design and the mention data should be a plain object. Hint: We might fix this with v2.0.0 f the mentions plugin.
+We design the API to accept an Immutable Map for a Mention. After saving your data structure to the server and using `convertFromRaw`, the mentions in there are plain objects. I (Nik) believe this is a law in our design and the mention data should be a plain object. Hint: we might fix this with v2.0.0 of the mentions plugin.
```

Comment:
Thanks for the background 🍻 

> I am wondering if allowing JavaScript objects wouldn't create the false assumption that the editor deals with JavaScript objects under the hood

If it helps, I'd say that callers shouldn't worry about what goes on under the hood as long as everything is properly encapsulated and the contracts are clear. I think it's fair to tell callers that you can pass in either a plain object or immutable object, but if a mention is passed to a callback then it will be an immutable object.
  
> The thing is that DraftJS is pretty tightly coupled to Immutable  

 True, and DraftJS works when using `convertFromRaw` so I imagine they're already doing the conversion.